src/server/ApiRoutes/Hold.js

```diff
@@ -556,7 +556,7 @@ function eddServer(req, res) {
     itemId,
     searchKeywords,
   } = req.body;
-  const searchKeywordsQuery = (searchKeywords) ? `&searchKeywords=${searchKeywords}` : '';
+  const searchKeywordsQuery = (searchKeywords) ? `&q=${searchKeywords}` : '';
```

Comment:
This change itself is fine but since you are updating the query string in other components to be`q=`, instead of `searchKeywords=`, then you need to update the code where that is read. In this file, line 286, you'll see 
```
searchKeywords: req.query.searchKeywords || '',
```
Since `searchKeywords` is no longer a query param after your change, the Store will only have `""` as the value for its `searchKeywords` field. Not updating to `req.query.q` can throw some bugs. I haven't verified this yet but thinking about it, it seems like you will need to make more changes to the PR.

It might be helpful for you if you also update the Store value to be `q` as well if you're making this change.

Reply:
@danamansana thoughts on changing the Store value from "searchKeywords" to "q"?

Reply:
As far as I can tell that is fine. How does this change affect the tests? Do we know all the behavior we need to check to make sure it is working correctly?

Reply:
Tests would need to be updated where appropriate. There are tests and components that pass down `searchKeywords` as a prop or call the Store using `store.getState().searchKeywords` and `store.state.searchKeywords` (note, the latter `store.state` is the incorrect way to access the data).